### DISCUSSION TITLE
Show commit message when browsing code

### DISCUSSION THREAD

**Discussion by thechnet** [community/community#3470] [Mobile]
Files should show the commit message of the most recent commit that modified them, just like im the browser.

**Comment by PranamBhat** [comment]
Can you please elaborated the issue/need?

**Reply by thechnet** [reply]
When browsing through files in the web version of GitHub, every directory and file shows the commit message of the most recent commit that modified it. This feature is missing in the mobile app.
The way this is implemented in the mobile web view is that the messages are only shown in landscape mode. I think it would make sense for the mobile app to reflect this behavior:
![image](https://user-images.githubusercontent.com/39877901/110023271-459f4c00-7d2d-11eb-8c20-337aa39a1a1d.jpeg)

**Comment by PranamBhat** [comment]
Hi @thechnet 

I got it! This is because of the screen resolution. 

Since it includes "files/folders + commit history + date/time it modified", it will be stretched for wider range.
